app/assets/stylesheets/admin/loans/timeline.scss

```diff
@@ -118,4 +118,15 @@
     margin: 0;
     vertical-align: middle;
   }
+
+  input[type="number"] {
+    max-width: 5em;
+    text-align: center;
+  }
+}
+
+#adjust-dates-modal {
```

Comment:
I'm curious why this block is not scoped within `.timeline` or similar. As it is, it is a global style declaration living at the bottom of a page-specific style file, which can lead to unintended consequences. Let's discuss.